atom/common/api/atom_bindings.cc

```diff
@@ -158,4 +161,25 @@ void AtomBindings::Log(const base::string16& message) {
   std::cout << message << std::flush;
 }
 
+void AtomBindings::CrashTest() {
+  Crash();
+}
+
+void AtomBindings::StartCrashReporter(const std::string productName, const std::string companyName, const std::string submitUrl, const std::string tmpPath, const std::map<std::string, std::string>& extra_parameters) {
+  
+  //std::cout << "Tmp Path: " << tmpPath << '\n';
+
+  std::map<std::string, std::string> allParameters;
+  allParameters.insert(std::make_pair("_productName", productName));
+  allParameters.insert(std::make_pair("_companyName", companyName));
+
+  for (auto& x: extra_parameters) {
+    allParameters.insert(std::make_pair(x.first, x.second));
```

Comment:
[nit] Use `allParameters.insert(extra_parameters.begin(), extra_parameterse.end())` instead of the loop